platform/security/filter/security-filter-web-sso/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -39,8 +39,4 @@
             interface="org.ops4j.pax.web.service.WebContainer"
             availability="mandatory"/>
 
-    <bean id="errorPage" class="org.codice.ddf.security.filter.websso.ErrorPage" init-method="registerErrorCodes">
-        <argument ref="webContainer"/>
-    </bean>
-
```

Comment:
✏️ We could technically remove the pax-web-api dependency in the pom as well. It was added for this ErrorPage stuff specifically: https://github.com/codice/ddf/commit/bb596ddbd3713ed9e8a87a46f8a32eddb4e2d2a3#diff-b6f9cda1762d1bbe6a83aa719ad6eb79